CHANGELOG.md

```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [21.04] - unreleased
+
+### Added
+- Create alerts in task dialog via graphql [#2425](https://github.com/greenbone/gsa/pull/2425) 
+- Added missing fields for getScanners query and parseObject() for scanner model [#2301](https://github.com/greenbone/gsa/pull/2301)
+
+### Changed
+
+### Fixed
+
+### Removed
+- Removed Edge <= 18 support [#2408](https://github.com/greenbone/gsa/pull/2408)
+- Removed Internet Explorer 11 support [#2399](https://github.com/greenbone/gsa/pull/2399)
+- Removed parsing and testing of textExcerpt [#2316](https://github.com/greenbone/gsa/pull/2316)
+- Removed extra parsing of comment and summary in Model [#2309](https://github.com/greenbone/gsa/pull/2309)
+- Removed severity class 'bsi' [#2434](https://github.com/greenbone/gsa/pull/2434)
```

Comment:
We are adding new entries to the top of the list in the changelog. Could you move the current entry accordingly, please?

Reply:
Done